app/src/main/java/com/eis/geoCalendar/gps/GPSPosition.java

```diff
@@ -1,36 +1,52 @@
 package com.eis.geoCalendar.gps;
 
+import android.location.Location;
+
 import androidx.annotation.NonNull;
 
 /**
  * Represents a position calculated with GPS latitude and longitude coordinates.
  * Adapter for {@link android.location.Location} class.
+ *
+ * @author someone from group 4
+ * @author Giorgia Bortoletti
  */
 public class GPSPosition {
 
-    public GPSPosition(float latitude, float longitude) {
-        //TODO: There can be more constructors
+    private Location position;
+
+    public GPSPosition(double latitude, double longitude) {
+        position = new Location("GPSPosition");
+        position.setLatitude(latitude);
+        position.setLongitude(longitude);
+    }
+
+    public GPSPosition(Location position){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4741777)